applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -339,6 +339,8 @@ namespace Kratos
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, Vector, STRESS_ON_NODE );
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, std::string, DESIGN_VARIABLE_NAME);
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, SERIAL_PARALLEL_EQUILIBRIUM_TOLERANCE);
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, PLASTIC_DAMAGE_PROPORTION);
```

Comment:
Please move it to claw if it is not used in structural

Reply:
All of them moved to the CLapp